x-pack/legacy/plugins/remote_clusters/public/app/sections/components/remote_cluster_form/remote_cluster_form.js

```diff
@@ -600,7 +646,9 @@ export class RemoteClusterForm extends Component {
           {this.renderSkipUnavailable()}
         </EuiForm>
 
-        {this.renderErrors()}
+        <div role="region" aria-live="assertive">
```

Comment:
I tried different options with the voice over and I had the best result witth

```html
<div role="region" aria-live="polite" aria-atomic="true">
```

Reply:
So I actually ended up removing this in favor of adding a `aria-describedby` to the button which I thought achieved similar but better (because you could navigate to the button after the fact and have the same thing read out to you again).

Let me know your thoughts!